checks/filenames.php

```diff
@@ -34,10 +34,11 @@ function check( $php_files, $css_files, $other_files ) {
 			'\.sql'              => __( 'SQL dump file', 'theme-check' ),
 			'__MACOSX'           => __( 'OSX system file', 'theme-check' ),
 			'\.lubith'           => __( 'Lubith theme generator file', 'theme-check' ),
-			'\.wie'               => __( 'Widget import file', 'theme-check' ),
-			'\.dat'               => __( 'Customizer import file', 'theme-check' ),
+			'\.wie'              => __( 'Widget import file', 'theme-check' ),
+			'\.dat'              => __( 'Customizer import file', 'theme-check' ),
 			'phpcs.xml.dist'     => __( 'PHPCS file', 'theme-check' ),
```

Comment:
This should be ammended to include other phpcs config files:
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#using-a-default-configuration-file

> Note: If multiple default configuration files are found, PHP_CodeSniffer will select one using the following order: `.phpcs.xml`, `phpcs.xml`, `.phpcs.xml.dist`, `phpcs.xml.dist`

Reply:
All of them will be found by the new xml check, is it necessary?

Reply:
The way it's currently written it will match only `.phpcs.xml.dist` and `phpcs.xml.dist`. But even the way it's written should be `phpcs\.xml\.dist` since `.` is a match for any character in a regex.

So we should add `phpcs\.xml`, since that will pick a part of `.phpcs.xml` (not the dot at the beginning).